ro.json

```diff
@@ -1476,8 +1476,8 @@
   "driving_plan": "Plan",
   "cookies_reject": "Respingere",
   "driving_arrival": "Sosire",
-  "driving_charging_alternative_chargers": "Find alternative charger",
-  "low_max_speed_warning": "You have set a low maximum speed value ({{speed}}) in setting.",
-  "charging_planned_charge_complete": "Planned charging completed",
-  "dismiss_rest_of_trip": "Rest of drive"
-}
\ No newline at end of file
+  "driving_charging_alternative_chargers": "Găsește încărcător alternativ",
+  "low_max_speed_warning": "Ai setat o valoare maximă scăzută a vitezei ({{viteza}}) în setare.",
```

Comment:
anything in {{FOO}} is a variable and shouldn't be translated, I will fix it and merge. Just letting you know @fasiol65